cluster-operator/src/main/java/io/strimzi/operator/cluster/operator/assembly/KafkaReconciler.java

```diff
@@ -764,7 +767,7 @@ private Map<String, String> podSetPodAnnotations(int nodeId) {
         podAnnotations.put(Ca.ANNO_STRIMZI_IO_CLUSTER_CA_CERT_GENERATION, String.valueOf(this.clusterCa.caCertGeneration()));
         podAnnotations.put(Ca.ANNO_STRIMZI_IO_CLUSTER_CA_KEY_GENERATION, String.valueOf(this.clusterCa.caKeyGeneration()));
         podAnnotations.put(Ca.ANNO_STRIMZI_IO_CLIENTS_CA_CERT_GENERATION, String.valueOf(this.clientsCa.caCertGeneration()));
-        podAnnotations.put(Annotations.ANNO_STRIMZI_LOGGING_APPENDERS_HASH, loggingHash);
+        podAnnotations.put(Annotations.ANNO_STRIMZI_LOGGING_APPENDERS_HASH, brokerLoggingHash.get(nodeId));
```

Comment:
Maybe now the `ANNO_STRIMZI_LOGGING_APPENDERS_HASH` name is not the right one anymore, because it's not just appenders?

Reply:
It is not, but I hope one day Kafka gets its act together and we can update it dynamically again in which case it would change back to _appenders only_. So I did not think it made sense to rename it. If you insist on it, I can do it, but otherwise I would keep it as it is.

Reply:
What do we say? The hope is the last one to die ;-)